src/update-by-tag.js

```diff
@@ -0,0 +1,41 @@
+const os = require('os')
+const ChildProcess = require('child_process')
+const { listHfDependencies } = require('./shared')
+
+const owner = 'bitfinexcom'
+
+const npm = (args) => {
+  console.log(args)
+  ChildProcess.spawnSync('npm', args, {
+    stdio: 'inherit',
+    shell: os.platform() === 'win32'
+  })
+}
+
+module.exports = (tag = 'canary') => {
+  if (!tag) {
```

Comment:
may be we should limit this to use only the `canary` tag to avoid mass updating the package if we do it by mistake.